interpreter/text/lexer.mll

```diff
@@ -339,6 +341,11 @@ rule token = parse
   | "offset" { OFFSET }
   | "import" { IMPORT }
   | "export" { EXPORT }
+  | "try" { TRY }
+  | "catch" { CATCH }
+  | "throw" { THROW }
+  | "rethrow" { RETHROW }
+  | "exception" { EXCEPTION }
```

Comment:
Ah, I meant only moving "exception" here, not the instruction names. Sorry for the ambiguity.